src/test/java/pl/zimi/zadanka/Zad4Test.java

```diff
@@ -0,0 +1,28 @@
+package pl.zimi.zadanka;
+
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+
+public class Zad4Test {
+    @Test
+    void checkstring() {
+
+        Zad4 object = new Zad4();
+
+        Assertions.assertEquals("blue is sky the", object.reverseString("the sky is blue"));
+
+    }
+    @Test
+    void checkspaces() {
+
+        Zad4 object = new Zad4();
+
+        Assertions.assertEquals("example good a", object.reverseString("a good   example"));
+
+    }
+    @Test
+    void Helloworld(){
```

Comment:
I detect that this code is problematic. According to the [Bad practice (BAD_PRACTICE)](https://spotbugs.readthedocs.io/en/stable/bugDescriptions.html#bad-practice-bad-practice), [Nm: Method names should start with a lower case letter (NM_METHOD_NAMING_CONVENTION)](https://spotbugs.readthedocs.io/en/stable/bugDescriptions.html#nm-method-names-should-start-with-a-lower-case-letter-nm-method-naming-convention).
Methods should be verbs, in mixed case with the first letter lowercase, with the first letter of each internal word capitalized.

Reply:
Ok, it will be changed.